packages/@react-spectrum/list/test/ListView.test.js

```diff
@@ -1231,523 +1223,4 @@ describe('ListView', function () {
       expect(grid.scrollTop).toBe(0);
     });
   });
-
-  describe('drag and drop', function () {
```

Comment:
Removed drag and drop into its own file so this file wouldn't end up like TableView's tests